src/plugins/api/package.json

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "api",
```

Comment:
So my thinking here was that index-patterns are a core part of Kibana (seeing as how kibana can't operate without one, and they're stored in the .kibana index). This is different than a plugin like Marvel, which stores its data in its own indices. I called the plugin `api` because I imagined it would be the location for future Kibana APIs (e.g. for visualizations or dashboards). Creating a separate plugin for each Kibana REST resource seems like overkill to me. Perhaps these endpoints should actually go in the kibana plugin? Then we could have endpoints like:

`/kibana/api/index-patterns`
`/kibana/api/visualizations`
`/kibana/api/dashboards`

Those match the marvel endpoints nicely e.g. `/marvel/api/v1/clusters`